e2e/e2e-create-windows-nodepool.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+source e2e-helper.sh
+
+log "Starting to create windows nodepool"
+
+# windowsNodepool="win19"
+# export windowsNodepool
+
+# az aks nodepool add --resource-group $RESOURCE_GROUP_NAME --cluster-name $CLUSTER_NAME --name $windowsNodepool --os-type Windows --node-count 1
+
+log "Created windows nodepool"
+out=$(az aks nodepool list --cluster-name $CLUSTER_NAME -g $RESOURCE_GROUP_NAME | jq '.[1].name')
+
+if [ "$out" == null ]; then
+    log "Creating windows nodepool"
+    az aks nodepool add --resource-group $RESOURCE_GROUP_NAME --cluster-name $CLUSTER_NAME --name $WINDOWS_NODEPOOL --os-type Windows --node-count 1
```

Comment:
Can we specify k8s version since some features may need the latest k8s version?

Reply:
If we need the latest k8s version, we need to specify k8s version for creating the cluster and windows nodepool. Besides, we need to modify the value of "customKubeBinaryURL" and "customKubeProxyImage". I will add some comments to notify it.

Reply:
@junjiezhang1997 Please help to create a task to support this.